org.eclipse.scanning.event/src/org/eclipse/scanning/event/ConsumerImpl.java

```diff
@@ -59,8 +61,8 @@
 
 final class ConsumerImpl<U extends StatusBean> extends AbstractQueueConnection<U> implements IConsumer<U> {
 
-	private static final Logger logger = LoggerFactory.getLogger(ConsumerImpl.class);
-	private static final long   ADAY   = 24*60*60*1000; // ms
+	private static final Logger LOGGER = LoggerFactory.getLogger(ConsumerImpl.class);
+	private static final long   ONE_DAY   = 24*60*60*1000; // ms
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "long". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)